### ISSUE TITLE
Windows 10 Templates broken on VS2017

### ISSUE THREAD

**Issue by DDReaper** [MonoGame/MonoGame#5515]
VS2017 is throwing template errors when creating a new Windows 10 project.

Open GL project is working and will check others.

Will submit a PR to fix soon

**Comment by Jjagg** [comment]
Gotta love maintaining templates...

**Comment by DDReaper** [comment]
So I got the rest of the templates to work and install. However, the Windows 10 template is foobared.
The old template wizard isn't available anymore and $Safename$ replacements aren't working (even though they work in a fresh template export)
Considering completely regenerating the template in 2017 now as it is becoming irksome.

**Comment by DDReaper** [comment]
Ok, completely confused now.  Just spotted the 3.6 release, installed and confirmed the Windows 10 template is FooBar'ed still.  However, what is strange, is that the original Template issue has gone (VS Complaining about the Wizard template version)
But it is not working as it doesn't generate the GUID's.

This does mean however, that Windows 10 on Vs2017 is not working.  Likely it will still work on 2015 but I haven't got that to test at the mo.

Is it work putting a post out on the MG blog for now @tomspilman ?

**Comment by DDReaper** [comment]
OK, just been doing some intensive testing and on at least three machines, Windows 10 no longer works at all, not even in Vs2015 now.  So it seems my "fixed" templates wren't being overwritten by the installer. After a clean uninstall and re-install produces the old error.

In 2015, you get the Wizard Template Failure, whereas 2017 just says Project creation failed (you have to add the project to an existing solution to get the same error as 2015)
This was the initial issue I was trying to fix, which was easy enough to resolve by upping the WinRT Wizard extension in the project template. 

However, this didn't work for Windows 10 at all which is where I got to.  Still testing but so far I'm struggling.  @dellis1972 any update from you (after an uninstall/reinstall of MG 3.6?)

**Comment by DDReaper** [comment]
Closed previous PR and will submit a new one.

In testing of the new 3.6 build, @dellis1972 rightly points out they are working if you have BOTH VS2015/VS2017 installed together (with all the correct options)

However, in testing I've found that with VS2017 alone, no matter your install options, you simply cannot create a new project.  Updating the WinRT template wizard to 15 (from 14) in the template definition does seem to fix it.

Also given both Dean's and my testing, it seems pointless to have Windows 8 templates as they don't work currently in VS2017.  There used to be an option but it seems they have retired the new templates for the Win 8 desktop / mobile platforms now in Vs2017.  So may update the installer to skip these.  The others all still work fine.

**Comment by davepermen** [comment]
Just got hit by this.. new sp4, vs2017 only, mg3.6 😢

**Comment by DDReaper** [comment]
For now @davepermen you can either grab the current dev build or simply edit your Windows 10 template and change the two references to:
```
Microsoft.VisualStudio.WinRT.TemplateWizards, Version=14.0.0.0
```
To
```
Microsoft.VisualStudio.WinRT.TemplateWizards, Version=15.0.0.0
```
And you will be back working.  Ignore the Win8 templates though

**Comment by DDReaper** [comment]
Fix applied in #5546 , closing
